src/Tribe/Views/V2/Template_Bootstrap.php

```diff
@@ -93,6 +93,25 @@ public function get_template_object() {
 			: tribe( Template\Event::class );
 	}
 
+	/**
+	 * Detemines wether we are in a Single event page or not,
+	 * base only on global context.
+	 *
+	 * @since  TBD
+	 *
+	 * @return bool
+	 */
+	public function is_single_event() {
+		$conditions = [
+			is_singular( TEC::POSTTYPE ),
+			'single-event' === tribe_context()->get( 'view' ),
+		];
+
+		$is_single_event = in_array( true, $conditions );
```

Comment:
Just use an `||` here and above? It's just 2 conditions.
Do you foresee more conditions being added?

Reply:
Yeah, my intention is to allow filtering + adding more conditionals. 
Something I am logging for after beta, with a couple other Single/Bootstrap fixes.